ja/lessons/ecto/querying.md

```diff
@@ -0,0 +1,443 @@
+---
+version: 1.0.0
```

Comment:
We may say this v1.0.1 https://github.com/elixirschool/elixirschool/pull/1734/files#diff-42858e1c36d5073e0f1f6c9b1b86d979 (It seems already fixed / no need to fix in Japanese..?

Reply:
Oops, I didn't realize that the version has increased. It looks like the change has only removed unnecessary spaces, so there is nothing we have to do but just bump up the version I guess.